cmd/baton-jira/config.go

```diff
@@ -5,13 +5,15 @@ import (
 )
 
 var (
-	jiraUrlField  = field.StringField("jira-url", field.WithRequired(true), field.WithDescription("Url to Jira service."))
-	emailField    = field.StringField("jira-email", field.WithRequired(true), field.WithDescription("Email for Jira service."))
-	apiTokenField = field.StringField("jira-api-token", field.WithRequired(true), field.WithDescription("API token for Jira service."))
+	jiraUrlField     = field.StringField("jira-url", field.WithRequired(true), field.WithDescription("Url to Jira service."))
+	emailField       = field.StringField("jira-email", field.WithRequired(true), field.WithDescription("Email for Jira service."))
+	apiTokenField    = field.StringField("jira-api-token", field.WithRequired(true), field.WithDescription("API token for Jira service."))
+	projectKeysField = field.StringField("jira-project-keys", field.WithDescription("Comma-separated list of Jira project keys to use for tickets."))
```

Comment:
This can be a field.StringSliceField(). Then you won't have to split on commas and stuff.

Reply:
Thanks! Fixed in [2def90c](https://github.com/ConductorOne/baton-jira/pull/25/commits/2def90c4166ae616a7f485eca284a3b63a59ec26)